src/main/scala/com/fulcrumgenomics/umi/DuplexConsensusCaller.scala

```diff
@@ -196,17 +197,21 @@ class DuplexConsensusCaller(override val readNamePrefix: String,
           val aQual = a.quals(i)
           val bQual = b.quals(i)
 
-          val (base, qual) = {
-            if      (aBase == NoCall || bBase == NoCall) (NoCall, PhredScore.MinValue)
-            else if (aBase == bBase) (aBase, (aQual + bQual).toByte)
+          // Capture the raw consensus base prior to masking it to N, so that we can compute
+          // errors vs. the actually called base.
+          val (rawBase, rawQual) = {
+            if      (aBase == bBase) (aBase, (aQual + bQual).toByte)
             else if (aQual > bQual)  (aBase, (aQual - bQual).toByte)
             else if (bQual > aQual)  (bBase, (bQual - aQual).toByte)
-            else (NoCall, PhredScore.MinValue)
+            else                     (aBase, PhredScore.MinValue)
           }
 
+          // Then mask it if appropriate
+          val (base, qual) = if (aBase == NoCall || bBase == NoCall || rawQual == PhredScore.MinValue) (NoCall, NoCallQual) else (rawBase, rawQual)
+
           bases(i)  = base
           quals(i)  = qual
-          errors(i) = min(sourceReads.count(s => s.bases(i) != NoCall && s.bases(i) != base), Short.MaxValue).toShort
+          errors(i) = min(sourceReads.count(s => s.length > i && s.bases(i) != NoCall && rawBase != NoCall && s.bases(i) != rawBase), Short.MaxValue).toShort
```

Comment:
The conditional is a lengthy one and took me a while to agree with.  Any chance you can move it out and comment well?